### PR TITLE
Changed default interface window size on mac

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/DEV-2477

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: libraries/ui/src/MainWindow.cpp [hunk 1]
```diff
@@ -62,7 +62,7 @@ void MainWindow::restoreGeometry() {
     // see http://doc.qt.io/qt-5/qsettings.html#restoring-the-state-of-a-gui-application
     QRect windowGeometry = QGuiApplication::primaryScreen()->availableGeometry();
 #if defined(Q_OS_MAC)
-    windowGeometry.setSize((windowGeometry.size() * 0.5f));
+    windowGeometry.setSize((windowGeometry.size() * 0.8f));
```
Comment (danteruiz) [inline libraries/ui/src/MainWindow.cpp]: Let not have this as a magic number.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22274/HighFidelity-Beta-PR16438-e5fcc57.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22274/HighFidelity-Quest-Beta-PR16438-e5fcc57.apk)

File: libraries/ui/src/MainWindow.cpp (hunk 1 above)
Comment (RebeccaStankus) [reply libraries/ui/src/MainWindow.cpp]: adding a variable

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22276/HighFidelity-Beta-PR16438-3f19bbf.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22276/HighFidelity-Quest-Beta-PR16438-3f19bbf.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22276/HighFidelity-Beta-PR16438-3f19bbf.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22276/HighFidelity-Beta-Interface-PR16438-3f19bbf.dmg

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22276/HighFidelity-Beta-Interface-PR16438-3f19bbf.exe

**Pull request merged**